docs/user/advanced.rst

```diff
@@ -482,8 +482,15 @@ To use HTTP Basic Auth with your proxy, use the `http://user:password@host/` syn
         "http": "http://user:pass@10.10.1.10:3128/",
     }
 
-Note that proxy URLs must include the scheme.
+To give a proxy for a specific scheme and host, use the
+`scheme://host` form for the key::
 
+    proxies = {
+      "http://10.20.1.128": "http://10.10.1.10:5323",
+    }
+
+Note that proxy URLs must include the scheme.
+    
```

Comment:
actually, it would work, since it's only scheme://hostname that is checked, not the port.  I'll still add a note.